modules/scylladb/scylladb.go

```diff
@@ -0,0 +1,159 @@
+		})
+
+		return nil
+	}
+}
+
+// WithCustomCommands sets custom commands with  values for the ScyllaDB container.
+// This is an option to overwrite the default commands with a custom one.
+// See more [here](https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/tips/best-practices-scylla-on-docker.html)
+func WithCustomCommands(cmds map[string]string) testcontainers.CustomizeRequestOption {
+	return func(req *testcontainers.GenericContainerRequest) error {
+		setCommandFlag(req, cmds)
+		return nil
+	}
+}
+
+// ConnectionHost returns the host and port of the Scylladb container with the given port
+// and obtaining the host and exposed port from the container
+// Eg: "host:port" -> "localhost:9042" -> "localhost:19042" -> "localhost:8000"
+func (c Container) ConnectionHost(ctx context.Context, port uint16) (string, error) {
+	host, err := c.Host(ctx)
+	if err != nil {
+		return "", fmt.Errorf("host: %w", err)
+	}
+
+	portStr := strconv.FormatInt(int64(port), 10)
+	containerPort, err := c.MappedPort(ctx, nat.Port(portStr))
+	if err != nil {
+		return "", fmt.Errorf("mapped port: %w", err)
+	}
+
+	return host + ":" + containerPort.Port(), nil
+}
+
+// Run starts a ScyllaDB container with the specified image and options
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*Container, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        img,
+		ExposedPorts: []string{port},
+		Cmd: []string{
+			"--developer-mode=1",
+			"--overprovisioned=1",
+			"--smp=1",
+			"--memory=512M",
+		},
+		WaitingFor: wait.ForAll(
+			wait.ForLog(".*initialization completed..*").AsRegexp(),
+			wait.ForListeningPort(port),
+			wait.ForExec([]string{"cqlsh", "-e", "SELECT bootstrapped FROM system.local"}).WithResponseMatcher(func(body io.Reader) bool {
+				data, _ := io.ReadAll(body)
+				return strings.Contains(string(data), "COMPLETED")
+			}),
+		),
+	}
+
+	genericContainerReq := testcontainers.GenericContainerRequest{
+		ContainerRequest: req,
+		Started:          true,
+	}
+
+	for _, opt := range opts {
+		if err := opt.Customize(&genericContainerReq); err != nil {
+			return nil, fmt.Errorf("customize: %w", err)
+		}
+	}
+
+	container, err := testcontainers.GenericContainer(ctx, genericContainerReq)
+	var c *Container
+	if container != nil {
+		c = &Container{Container: container}
+	}
+
+	if err != nil {
+		return c, fmt.Errorf("generic container: %w", err)
+	}
+
+	return c, nil
+}
+
+func setCommandFlag(req *testcontainers.GenericContainerRequest, flag map[string]string) {
+	cmds := make([]string, 0, len(req.Cmd)+len(flag))
+
+	for _, cmd := range req.Cmd {
+		seps := strings.SplitN(cmd, "=", 1)
+		val, ok := flag[seps[0]]
+
+		if !ok {
+			cmds = append(cmds, cmd)
+		} else {
+			cmds = append(cmds, fmt.Sprintf("%s=%s", seps[0], val))
+			delete(flag, seps[0])
+		}
+	}
+
+	for key, val := range flag {
+		cmds = append(cmds, fmt.Sprintf("%s=%s", key, val))
```

Comment:
suggestion: no need for sprintf here just add
```suggestion
		cmds = append(cmds, key+"="+val)
```